### ISSUE TITLE
Youtube ads playing despite adblock being enabled

### ISSUE THREAD

**Issue by viasux** [bromite/bromite#2353]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser; not the website nor F-Droid nor anything else.

### Can the bug be reproduced with corresponding Chromium version?

Yes

### Bromite version

105.0.5195.41

### Device architecture

arm64

### Android version

12.1

### Device model

Google Pixel 4a5g

### Changed flags

no flags changed

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

Yes

### Is this bug a crash?

No.

### Describe the bug

Youtube ads still play while adblock is enabled. I understand that it is possible to add custom adblock rules, however youtube is a very popular site and I believe ads should be blocked on it by default. Is this possibly because of something to do with manifest v3?

### Steps to reproduce the bug

1. Go to youtube.com
2. Watch a video
3. See ads

### Expected behavior

I expect ads to not show on youtube.com

### Screenshots

![image](https://user-images.githubusercontent.com/47131867/192371240-16cac4d8-95fc-49d2-84ca-a7655d73ec88.png)

**Comment by kokukyocho** [comment]
Hello @viasux!

I am a bot :robot:

Thanks for submitting this issue!
Since this bug is reproducible with Chromium please report it upstream on the [Chromium issue tracker](https://bugs.chromium.org/p/chromium/issues/list) and do not forget to read [Chromium project bug reporting guidelines](https://www.chromium.org/for-testers/bug-reporting-guidelines) first.

**Comment by viasux** [comment]
> Hello @viasux!
> 
> I am a bot 🤖
> 
> Thanks for submitting this issue! Since this bug is reproducible with Chromium please report it upstream on the [Chromium issue tracker](https://bugs.chromium.org/p/chromium/issues/list) and do not forget to read [Chromium project bug reporting guidelines](https://www.chromium.org/for-testers/bug-reporting-guidelines) first.

Chromium does not have ad-blocking capability, thus it being reproducible is irrelevant :p
